GW2EIEvtcParser/EIData/ProfHelpers/Necromancer/ScourgeHelper.cs

```diff
@@ -10,17 +10,18 @@ namespace GW2EIEvtcParser.EIData
 {
     internal static class ScourgeHelper
     {
-
         internal static readonly List<InstantCastFinder> InstantCastFinder = new List<InstantCastFinder>()
         {
             // Trail of Anguish ? Unique effect?
             new EffectCastFinder(TrailOfAnguish, EffectGUIDs.ScourgeTrailOfAnguish).UsingSrcSpecChecker(Spec.Scourge).UsingICD(6100),
```

Comment:
If you found a non PoV only buff for trails of anguish, it is better to use it for the InstantCast detection, buff based solutions are always the best.

Reply:
The 42311 one seems to be the only non-POV-only out of them, but for some reason it has twice the duration (12s vs 6s). I'll include it for now.